src/templates/partials/theader.html

```diff
@@ -46,7 +46,7 @@
                     <a class="navbar-link">View As</a>
                     <div class="navbar-dropdown">
                         <a href="#" class="navbar-item">User</a>
-                        <a href="#" class="navbar-item">Organization 1</a>
+                        <a href="{% url 'donor_dashboard:org_list' %}" class="navbar-item">Organization 1</a>
```

Comment:
Can we change "Organization 1" to just "Organisation"?

Reply:
I can handle this on front-end as well

Reply:
Yes, sounds good!

Reply:
I have updated this part as well. Please take a look.